src/vm/genericdict.h

```diff
@@ -130,22 +149,23 @@ class DictionaryLayout
     // another structure (e.g. MethodTable)
     static DWORD GetFirstDictionaryBucketSize(DWORD numGenericArgs, PTR_DictionaryLayout pDictLayout);
```

Comment:
There are no buckets anymore after your change. This method should be renamed.

Reply:
Renamed to GetDictionarySizeFromLayout